eng/Directory.Packages.props

```diff
@@ -142,8 +142,8 @@
     -->
     <PackageVersion Include="Microsoft.VisualStudio.LanguageServer.Client.Implementation" Version="17.10.72-preview" />
     <PackageVersion Include="NuGet.ProjectModel" Version="6.8.0-rc.112" />
-    <PackageVersion Include="Microsoft.TestPlatform.TranslationLayer" Version="$(_MicrosoftTestPlatformVersion)" />
-    <PackageVersion Include="Microsoft.TestPlatform.ObjectModel" Version="$(_MicrosoftTestPlatformVersion)" />
```

Comment:
`_MicrosoftTestPlatformVersion` property seems to be unused now, should it be removed?

Reply:
Fixed with https://github.com/dotnet/roslyn/pull/77179/commits/f252374fd5b6fb2c1e7cd223da9c4f28562160d8